package.json

```diff
@@ -15,10 +15,11 @@
     "url": "https://github.com/danawoodman/react-fontawesome/issues"
   },
   "main": "lib/index.js",
+  "style": "lib/react-fontawesome.css",
   "scripts": {
     "build": "babel $npm_package_config_entry --out-dir $npm_package_config_output",
     "dist": "npm run lint && npm run build && npm test && npm run docs",
-    "docs": "jsdoc2md $npm_package_config_mainFile > api.md",
+    "docs": "node node_modules/.bin/jsdoc2md $npm_package_config_mainFile > api.md",
```

Comment:
This should not have the node_modules bit since `package.json` includes anything in `node_modules/.bin` in the execution environment when you call npm tasks